docs/datamodel/links.rst

```diff
@@ -472,8 +473,8 @@ properties of the relationships and properties of the target objects. On the
 other hand, for one-to-one, one-to-many, and many-to-one relationships there's
 an exact correspondence between the link and one of the objects being linked.
 In these situations any property of the relationship can be equally expressed
-as the property of the target object (for one-to-many and one-to-one cases) or
-as the property of the source object (for many-to-one and one-to-one cases).
+as the property of the source object (for one-to-many and one-to-one cases) or
+as the property of the target object (for many-to-one and one-to-one cases).
```

Comment:
I though these should be switched, unless I am misunderstanding something.

Reply:
I think so...@raddevon can you confirm? Feels like maybe a chart could be useful here as I feel a bit dizzy reading that paragraph.

Reply:
This change looks correct to me!